plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/persistence/LibraryClasspathContainerSerializer.java

```diff
@@ -44,6 +47,8 @@
 @Creatable
 public class LibraryClasspathContainerSerializer {
 
+  private static final String APPENGINE_LIBRARIES_BUNDLE_NAME = "com.google.cloud.tools.eclipse.appengine.libraries";
```

Comment:
Seems like this is used only by `LibrariesBundleStateLocationProvider`. Put it there?